### PR TITLE
Update User Guide documentation

### PR DESCRIPTION
**Pull request by marcus-pzj**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -232,7 +238,7 @@ Format: `undo`
 
 [NOTE]
 ====
-Some commands are undoable. e.g. `wipe PlanMySem`.
+Some commands are undoable. e.g. `clear`.
```
Comment (dingheng4448) [inline docs/UserGuide.adoc]: I think we should specify all commands that are undoable or the user might not know which other commands are undoable.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -18,30 +18,34 @@ By: `Team T08-3`      Since: `Jan 2019`      Licence: `MIT`
 
 == Introduction
 
-PlanMySem is a CLI (Command Line Interface) scheduling/calendar application that targets NUS students and staff who prefer to use a desktop application for managing their schedule/calendar.
-PlanMySem automatically creates a planner that is synchronised according to the NUS academic calendar for the current semester and enables easy creation, editing and deleting of items.
+Thank you for using _PlanMySem_!
+
+_PlanMySem_ is a CLI (Command Line Interface) scheduling/calendar application that targets NUS students and staff who prefer to use a desktop application for managing their schedule/calendar.
```
Comment (dingheng4448) [inline docs/UserGuide.adoc]: Perhaps we could change the first line to "PlanMySem is a text-based (Command Line Interface) scheduling/calendar application ...."

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -18,30 +18,34 @@ By: `Team T08-3`      Since: `Jan 2019`      Licence: `MIT`
 
 == Introduction
 
-PlanMySem is a CLI (Command Line Interface) scheduling/calendar application that targets NUS students and staff who prefer to use a desktop application for managing their schedule/calendar.
-PlanMySem automatically creates a planner that is synchronised according to the NUS academic calendar for the current semester and enables easy creation, editing and deleting of items.
+Thank you for using _PlanMySem_!
```
Comment (dingheng4448) [inline docs/UserGuide.adoc]: Is it weird to already thank the audience for using our application? Or should we just write "Welcome to PlanMySem!"?

**Review by dingheng4448 — COMMENTED** [review]
Good updates to the user guide!

File: docs/UserGuide.adoc (hunk 1 above)
Comment (marcus-pzj) [reply docs/UserGuide.adoc]: Actually, only the clear command is undoable so far. I'll make that clear.

File: docs/UserGuide.adoc (hunk 2 above)
Comment (marcus-pzj) [reply docs/UserGuide.adoc]: That's better, thanks.

File: docs/UserGuide.adoc (hunk 3 above)
Comment (marcus-pzj) [reply docs/UserGuide.adoc]: Alright, will make the changes.

**Pull request merged**